ajax/libs/jsrsasign/7.1.2/jsrsasign-all-min.js

```diff
@@ -0,0 +1,289 @@
+/*
+ * jsrsasign(all) 7.1.3 (2017-04-16) (c) 2010-2017 Kenji Urushima | kjur.github.com/jsrsasign/license
```

Comment:
Yes, that's what I usually do, sorry about getting the wrong file, I can't remember why I got the wrong file of 7.1.2. I am fixing it.